server/sandbox_run_linux.go

```diff
@@ -450,7 +450,7 @@ func (s *Server) runPodSandbox(ctx context.Context, req *types.RunPodSandboxRequ
 		g.ClearProcessCapabilities()
 		capabilities.AddCapabilities = append(capabilities.AddCapabilities, s.config.DefaultCapabilities...)
```

Comment:
do we still need this line if we're adding them in setupCapabilities?

Reply:
Yeah, need to drop this. Fixing.

Reply:
done